src/test/java/bdd/steps/IndividualRow.java

```diff
@@ -0,0 +1,33 @@
+package bdd.steps;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.digital.ho.pttg.dto.Individual;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+import static java.time.format.DateTimeFormatter.ISO_DATE;
+
+@AllArgsConstructor
+@Getter
```

Comment:
Since all the step implementations have been rewritten as Java too, might this be a good opportunity to change the getters to fluent?  We're no longer tied to the Groovy getX() construct

Reply:
Good point. I'll do that in the next commit